jetty-home/src/main/resources/bin/jetty.sh

```diff
@@ -492,16 +492,13 @@ case "$ACTION" in
 
         touch "$JETTY_PID"
         chown "$JETTY_USER" "$JETTY_PID"
-        # FIXME: Broken solution: wordsplitting, pathname expansion, arbitrary command execution, etc.
         su - "$JETTY_USER" $SU_SHELL -c "
           cd \"$JETTY_BASE\"
           echo ${RUN_ARGS[*]} start-log-file=\"$JETTY_START_LOG\" | xargs ${JAVA} > /dev/null &
-          disown \$!
-          echo \$! > \"$JETTY_PID\""
+          disown $(pgrep -P $!)"
+        pgrep
```

Comment:
I need to get rid of this line

Reply:
Note that I saw a previous comment about the disown only affecting the xargs.   However, I think that is enough as the following experiment shows:
```sh
# sleep 100 | sleep 100 &
[1] 588261

# jobs
[1]+  Running                 sleep 100 | sleep 100 &

[] disown %1

# jobs

# ps
    PID TTY          TIME CMD
 587972 pts/1    00:00:00 bash
 588260 pts/1    00:00:00 sleep
 588261 pts/1    00:00:00 sleep
 588317 pts/1    00:00:00 ps
```

Reply:
@joakime but the disown is not about killing.   This PR is creating the pid file so that the killing is done correctly.
The disown is about breaking the link between the shell starting jetty and the jetty process itself.   I think my demo shows that disowning the $! pid is enough to break that link.